tests/unit_tests/test_all_icons.py

```diff
@@ -278,7 +278,7 @@ def test_all_icon_metadata_endpoint(self):
                         self.assertIsInstance(
                             fraction, (int, float), msg='"anchor" fraction should be int or float'
                         )
-                        self.assertTrue(fraction > 0, msg='"anchor" fraction should be > 0')
+                        self.assertTrue(fraction >= 0, msg='"anchor" fraction should be >= 0')
```

Comment:
Just wondering if we should not here have a better checker here instead of the for loop:

```python
if json_response['name'] in ICON_ANCHORS:
    self.assertListEqual(json_response['anchor'], ICON_ANCHORS[json_response['name']], msg=f'Wrong anchor for {json_response["name"]}')
else:
    self.assertListEqual(json_response['anchor'], [0.5, 0.5], msg=f"{json_response['name']} anchor is not in the middle bottom"
```

Just for my understanding the default anchor seems to `[0.5, 0.5]` and seems to be at the middle bottom of the icon, is that correct ? How the anchor values are used in the viewer ? Having `[0.5, 0.5]` I would have expected to be in the middle ? Might be also a good idea to enhanced the doc-api-specs with a better description of the origin of the offset used for the anchor.

Reply:
from what I understood while changing that, it is a fraction value of the size of the icon, from 0.0 to 1.0 (see https://openlayers.org/en/latest/apidoc/module-ol_style_Icon-Icon.html)